cnf/ext/repositories.bnd

```diff
@@ -11,6 +11,23 @@ eclipse-orbit: aQute.bnd.repository.osgi.OSGiRepository;\
  locations=http://download.eclipse.org/tools/orbit/downloads/drops/R20150124073747/repository/index.xml.gz; \
  poll.time=-1; \
  cache=${build}/cache/stable/EclipseOrbit
+eclipse-m2e: aQute.bnd.repository.p2.provider.P2Repository; \
+ name="Eclipse m2e 1.6.2"; \
+ url="http://download.eclipse.org/technology/m2e/releases/1.6/1.6.2.20150902-0002/"
+eclipse-emf: aQute.bnd.repository.p2.provider.P2Repository; \
+ name="Eclipse EMF Releases"; \
+ url="http://download.eclipse.org/modeling/emf/updates/releases/"
```

Comment:
I looked at this we can remove most of these bundles if we want to stripped down test launch.  Lets do that.